### ISSUE TITLE
Ability to reboot via integration

### ISSUE THREAD

**Issue by Ucsus** [dbuezas/icsee-ptz#33]
Hi, David! Thanx for the integration!
Is it possible to programmatically reboot the camera? From time to time, the image from the cameras disappears (most likely when they lose connection with Wi-Fi) and only a reboot in the XMEye application helps.

**Comment by dbuezas** [comment]
No, there isn't. I don't know if it can be implemented (probably)

If the option is there in xmeye, try changing the codec
